src/cdk/overlay/overlay.ts

```diff
@@ -16,13 +16,15 @@ import {
   Injectable,
   Injector,
   NgZone,
+  Optional,
 } from '@angular/core';
 import {OverlayKeyboardDispatcher} from './keyboard/overlay-keyboard-dispatcher';
 import {OverlayConfig} from './overlay-config';
 import {OverlayContainer} from './overlay-container';
 import {OverlayRef} from './overlay-ref';
 import {OverlayPositionBuilder} from './position/overlay-position-builder';
 import {ScrollStrategyOptions} from './scroll/index';
+import {ANIMATION_MODULE_TYPE} from '@angular/platform-browser/animations';
```

Comment:
This introduces a dependency on `@angular/animations` that I don't think we want in the CDK

Reply:
Isn’t the token technically in platform-browser? Also we’ve got the experimental/dialog which has a dependency on animations.

Reply:
It is, though I assume `@angular/platform-browser/animations` would depend on `@angular/animations`? If it doesn't then maybe this is fine. @jelbourn might know more